examples/05_stable_diffusion/src/benchmark.py

```diff
@@ -56,7 +63,17 @@ def benchmark_unet(
     verify=False,
 ):
 
-    exe_module = Model("./tmp/UNet2DConditionModel/test.so")
+    """
+    Set the OS environment variable AITEMPLATE_WORK_DIR to point to an absolute
+    path to a directory which has AITemplate compiled artifacts the model(s). 
+    Make sure the OS user running this script has read and write permissions to 
+    this directory. By default, the it will look for compiled artifacts under 
+    tmp/ folder of the current working directory. 
+    """
+
+    file_name = get_file_location_Unet()
```

Comment:
Nit, maybe we can avoid having capital letters in function names?

Reply:
done